### ISSUE TITLE
the optimal hyperplane

### ISSUE THREAD

**Issue by laoruan100** [Doctorado-ML/STree#54]
Hi, thank you so much for the package! 
How to know the optimal hyperplane for each node (except leaf nodes)? 
And is it possible to focus on the classification accuracy of a certain class of samples (for example, I want more 0 class samples to be accurately classified instead of getting a higher overall accuracy)?

**Comment by rmontanana** [comment]
Hi!

You have a SVM classifier in each node and you can ask it for the hyperplane ([see coef_](https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html)). To see if the node is a leave you can ask the node for it.

The second question is a bit more complicated, I don't know if I understand well the question, but I think you can try the samples weights to achieve what you want.

Cheers

**Comment by laoruan100** [comment]
Thank you!
